setup.py

```diff
@@ -9,17 +9,18 @@
     license='MIT',
     python_requires='>=3.6',
     description='ASCII-chart version of how Bitcoin grows',
-    long_description='''A console application that allows you to see a graph of the Bitcoin price change in any time interval. Also prints the current cost and comparison with yesterday's value.
-Almost all currencies are supported.
-More info: https://github.com/gabolaev/crypterma/blob/master/README.md''',
-    packages=['crypterma','crypterma.configs'],
+    long_description='''A console application that allows you to see a graph of the Bitcoin price change in any time interval.
```

Comment:
to avoid ident shit like that, we can alter such lines into:
``` python
long_description=(
    'A console application that allows you to see a graph '
    'of the Bitcoin price change in any time interval. Also '
    'prints the current cost and comparison with yesterday\'s value.'
    'Almost all currencies are supported.'
    'More info: https://github.com/gabolaev/crypterma/blob/master/README.md'
)
```
result:
``` python
❯ ipython
Python 3.7.0 (default, Jun 29 2018, 20:13:13)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: long_description=(
   ...:     'A console application that allows you to see a graph '
   ...:     'of the Bitcoin price change in any time interval. Also '
   ...:     'prints the current cost and comparison with yesterday\'s value.'
   ...:     'Almost all currencies are supported.'
   ...:     'More info: https://github.com/gabolaev/crypterma/blob/master/README.md'
   ...: )

In [2]: long_description
Out[2]: "A console application that allows you to see a graph of the Bitcoin price change in any time interval. Also prints the current cost and comparison with yesterday's value.Almost all currencies are supported.More info: https://github.com/gabolaev/crypterma/blob/master/README.md"

In [3]:
```

Reply:
Done.